lib/suspenders/app_builder.rb

```diff
@@ -209,8 +209,8 @@ def configure_rspec
       copy_file "spec_helper.rb", "spec/spec_helper.rb"
     end
 
-    def configure_travis
-      template 'travis.yml.erb', '.travis.yml'
+    def configure_ci
+      template 'circle.yml.erb', 'circle.yml'
```

Comment:
It automatically uses your `.ruby-version`, is there any reason we should specify it twice? That seems like it would just make it harder to upgrade.


I left it as an erb just in case we ended up adding dynamic information, happy to remove it though, I don't feel strongly either way.